common/persistence/dataInterfaces.go

```diff
@@ -955,68 +948,31 @@ type (
 		NotificationVersion int64
 	}
 
-	// MutableStateStats is the size stats for MutableState
-	MutableStateStats struct {
-		// Total size of mutable state
-		MutableStateSize int
+	// MutableStateStatus is the size stats for MutableState
```

Comment:
I'd have preferred the original name for this struct ... it's more Statistics than Status.  Or just go with MutableStateStatistics.  And then your vars can be (e.g.) MutableStateStats

Reply:
echo that.

Reply:
sure